src/aznfswatchdog

```diff
@@ -176,6 +182,22 @@ while :; do
 
     epoch_now=$(date +%s)
 
+    # To check if any AZNFS updates available.
+    check_for_latest_update=false
+    if [ "$AKS_USER" != "true" ] && [ $epoch_now -ge $next_update_detection_epoch ]; then
+        check_for_latest_update=true
+        next_update_detection_epoch=$(expr $(date +%s) + $UPDATE_DETECTION_FREQUENCY)
+    fi
+
+    if $check_for_latest_update; then
+        if [ -e "/opt/microsoft/aznfs/aznfs_install.sh" ]; then
```

Comment:
Add a var for this file as well in common.sh and use that shorten var everywhere.